### PR TITLE
Make home page better & have Infrastructure guidance

### PR DESCRIPTION
**Pull request by kjdchapman**
Made the home page steps clearer and link into the relevant tools for setting things up:

image

The infrastructure guidance is **very** WIP at the moment - linking into tech-docs from here felt _so bad_, I felt compelled to not do it:

image

### CODE REVIEW

**Comment by kjdchapman** [comment]
Notably this adds _yet another_ tab to the IPs page - but think it's better to do the separation in another PR rather than muddy this one.

**Comment by kjdchapman** [comment]
Updated screenshot of the 'Your Infrastructure' tab - I'm almost happy with it now!

image

File: app/views/ips/index.html.erb [hunk 1]
```diff
@@ -9,78 +9,115 @@
...
+    </p>
+
+    <h3 class="govuk-heading-m"> Create an SSID </h3>
+    <ul class="govuk-list govuk-list">
+      <li>Name: <strong>GovWifi</strong></li>
+      <li>Type: <strong>WPA2-Enterprise (AES encryption)</strong></li>
+      <li>Inner Encryption: <strong>MsChapV2</strong> </li>
+    </ul>
+
+    <!-- <h3 class="govuk-heading-m"> Protect your users? </h3>
```
Comment (adrianclay) [inline app/views/ips/index.html.erb]: Commented code. 😨

Comment (kjdchapman) [reply app/views/ips/index.html.erb]: I agree, let's kill this...

**Pull request merged**